buildpacks/evergreen_function/buildpack.toml

```diff
@@ -0,0 +1,23 @@
+api = "0.2"
+
+[buildpack]
+id = "evergreen/fn"
+version = "0.1"
+name = "Evergreen Function"
+
+[[order]]
+  [[order.group]]
+    id = "heroku/nodejs-engine"
+    version = "0.4.3"
+
+  [[order.group]]
+    id = "heroku/nodejs-npm"
+    version = "0.2.0"
+
+  [[order.group]]
+    id = "io.projectriff.node"
+    version = "0.3.1-BUILD-SNAPSHOT"
```

Comment:
Is there a non-snapshot release of this?

Reply:
@jabrown85 I believe there is now with the changes that Ben mentioned.  We just need to re-visit how Ben's changes will affect the other buildpacks with the renaming of `[provides]` and `[requires]` stuff, and we'll need to also the `http-streaming-adapter` buildpack to the list.   I did add a spike in the backlog (not in this Sprint, but can still prioritize for this Sprint if we want).

https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B000000893wxIAA/view